code/src/HappyForm.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react'
+import './HappyForm.css'
+
+const Happyform = "https://happy-thoughts-technigo.herokuapp.com/thoughts";
```

Comment:
This is not the best name for url variable :

1. First of all, only components should have names written in uppercase
2. Something like `MESSAGE_URL` would be far better